pallets/teerex/src/lib.rs

```diff
@@ -640,6 +650,8 @@ pub mod pallet {
 		TooManyTopics,
 		/// The length of the `data` passed to `publish_hash` exceeds the limit.
 		DataTooLong,
+		/// The number of enclave exceed the upper limmit.
+		ExceedEnclaveNumber,
```

Comment:
Maybe `TooManyEnclaves` to be consistent with other errors, or `ExceedMaxEnclaveCount`?